pkg/controllers/providers/instancetype/capacity/suite_test.go

```diff
@@ -75,7 +79,9 @@ var _ = BeforeSuite(func() {
 	nodeClass = test.EC2NodeClass()
 	nodeClaim = coretest.NodeClaim()
 	node = coretest.Node()
-	controller = controllersinstancetypecapacity.NewController(env.Client, awsEnv.InstanceTypesProvider)
+	cloudProvider := cloudprovider.New(awsEnv.InstanceTypesProvider, awsEnv.InstanceProvider, events.NewRecorder(&record.FakeRecorder{}),
```

Comment:
Do you think that this means that we should gen the CloudProvider inside of the `awsEnv`?

Reply:
Discussed offline, I agree and we'll take it as a fast follow.